tests/integration/common/checks-after-conversion/test_verify_strings_in_log.py

```diff
@@ -1,7 +1,5 @@
 import re
 
-from test_helpers.common_functions import log_file_data
-
```

Comment:
Do not remove this line. It is needed in the test

Reply:
Couldn't we just move it to conftest to have it be available without import? It doesn't feel good to have to import a fixture like this as it is a bit confusing. I'd rather have pytest take care of it in something like conftest
```diff
diff --git a/tests/integration/conftest.py b/tests/integration/conftest.py
index bfb0596b..f299bc1c 100644
--- a/tests/integration/conftest.py
+++ b/tests/integration/conftest.py
@@ -24,12 +24,12 @@ except ImportError:
 #
 pytest.register_assert_rewrite("test_helpers")
 
-from test_helpers.common_functions import SystemInformationRelease, get_full_kernel_title  # noqa: E402
-from test_helpers.satellite import Satellite  # noqa: E402
-from test_helpers.shell import live_shell  # noqa: E402
-from test_helpers.subscription_manager import SubscriptionManager  # noqa: E402
-from test_helpers.vars import SYSTEM_RELEASE_ENV, TEST_VARS  # noqa: E402
-from test_helpers.workarounds import workaround_grub_setup  # noqa: E402
+from test_helpers.common_functions import SystemInformationRelease, get_full_kernel_title
+from test_helpers.satellite import Satellite
+from test_helpers.shell import live_shell
+from test_helpers.subscription_manager import SubscriptionManager
+from test_helpers.vars import SYSTEM_RELEASE_ENV, TEST_VARS
+from test_helpers.workarounds import workaround_grub_setup
 
 
 logging.basicConfig(level=os.environ.get("DEBUG", "INFO"), stream=sys.stderr)
@@ -538,3 +538,20 @@ def backup_directory(shell, request):
     yield backup_path
 
     shell(f"rm -rf {backup_path}")
+
+
+@pytest.fixture()
+def log_file_data():
+    """
+    Helper fixture.
+    Reads and returns data from the convert2rhel.log file.
+    Mainly used for after conversion checks, where we match required strings to the log output.
+    """
+    convert2rhel_log_file = "/var/log/convert2rhel/convert2rhel.log"
+
+    assert os.path.exists(convert2rhel_log_file), "The c2r log file does not exits."
+
+    with open(convert2rhel_log_file, "r") as logfile:
+        log_data = logfile.read()
+
+        return log_data
diff --git a/tests/integration/test_helpers/common_functions.py b/tests/integration/test_helpers/common_functions.py
index 069626b1..c2cd9e07 100644
--- a/tests/integration/test_helpers/common_functions.py
+++ b/tests/integration/test_helpers/common_functions.py
@@ -108,20 +108,3 @@ def get_custom_repos_names():
                 f"rhel-{system_version.major}-for-x86_64-appstream-rpms",
             ]
     return repos
-
-
-@pytest.fixture()
-def log_file_data():
-    """
-    Helper fixture.
-    Reads and returns data from the convert2rhel.log file.
-    Mainly used for after conversion checks, where we match required strings to the log output.
-    """
-    convert2rhel_log_file = "/var/log/convert2rhel/convert2rhel.log"
-
-    assert os.path.exists(convert2rhel_log_file), "The c2r log file does not exits."
-
-    with open(convert2rhel_log_file, "r") as logfile:
-        log_data = logfile.read()
-
-        return log_data
```